compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/Dialog.desktop.kt

```diff
@@ -262,12 +267,13 @@ fun DialogWindow(
     visible: Boolean = true,
     title: String = "Untitled",
     icon: Painter? = null,
-    decoration: WindowDecoration,
+    decoration: WindowDecoration = WindowDecoration.SystemDefault,
```

Comment:
We have it experimental for some time. We need to start stabilize functions which we added earlier, and schedule stabilizing `modalityType` as well. Please, create issues for stabilizing decoration/modalityType with appropriate target versions ([this doc](https://docs.google.com/document/d/15vffH9b_8Jqy4I9RK103HgCwr1c8lhhm1MGT7pvfT8w/edit?tab=t.0) says we usually should stabilize in the +2 version)

Reply:
Created https://youtrack.jetbrains.com/issue/CMP-8730/Remove-experimental-status-from-DialogWindow-and-SwingDialog, but there's no 1.12 target version to set.

Reply:
Added new versions with discussion with Kropp